CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## Unreleased
 
+* Enable [OAuth2 PKCE extension](https://datatracker.ietf.org/doc/html/rfc7636) in our GDS OAuth2 OmniAuth Strategy.
```

Comment:
I think it might be worth referencing the corresponding change in Signon and making it clear that this gds-sso change will be compatible with Signon.

Reply:
Minor: I'd change "our GDS OAuth2 OmniAuth Strategy" -> "the GDS OAuth2 OmniAuth strategy".

Reply:
Done x 2 :+1: